ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -477,13 +477,18 @@ objects:
         cp -r /var/lib/openshift-install/upi/metal/* ${TMP_SHARED}/terraform/
         cp /bin/terraform-provider-matchbox ${TMP_SHARED}/terraform/
 
-        BASE_URI="$(jq -r '.baseURI' /var/lib/openshift-install/rhcos.json | sed 's|https://|http://|' | sed 's|rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com|releases-art-rhcos.svc.ci.openshift.org|')"
+        if openshift-install coreos print-stream-json 2>/tmp/err.txt >/var/lib/openshift-install/coreos-print-stream.json; then
+                RHCOS_JSON_FILE="/var/lib/openshift-install/coreos-print-stream.json"
+        else
+                RHCOS_JSON_FILE="/var/lib/openshift-install/rhcos.json"
+        fi
+        BASE_URI="$(jq -r '.baseURI' "${RHCOS_JSON_FILE}" | sed 's|https://|http://|' | sed 's|rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com|releases-art-rhcos.svc.ci.openshift.org|')"
```

Comment:
Yeah the jq paths are different for both files. You might need to set the environment variables twice differently in both the if conditions. The coreos-print-stream path is more like .architectures.x86_64.artifacts.metal.pxe if I am not wrong.

Reply:
I concur. For example, there is no baseURI in the coreos-print-stream.

Reply:
Thanks. I had no idea the files were so different. I have updated the logic. PTAL.